package.json

```diff
@@ -31,6 +32,7 @@
     "nanoid": "^3.1.25",
     "pet-pet-gif": "^1.0.9",
     "profanity-matcher": "^1.0.1",
+    "request-promise-native": "^1.0.8",
```

Comment:
We use axios for web requests at the minute, let's not add additional request packages for now.